templates/vmset/cloudinit/userdata

```diff
@@ -12,4 +12,8 @@ chpasswd:
 {{- end }}
 bootcmd:
   - "mkdir /mnt/kubeconfig"
-  - "mount /dev/disk/by-id/virtio-fencingdisk /mnt/kubeconfig"
\ No newline at end of file
+  - "mkdir /root/kubeconfig"
+  - "mount /dev/disk/by-id/virtio-fencingdisk /mnt/kubeconfig"
+  - "chown -R root:root /mnt/kubeconfig"
+  - "chmod -R 0600 /mnt/kubeconfig"
+  - "cp /mnt/kubeconfig/* /root/kubeconfig/."
```

Comment:
bootcmd will run on every boot, so we should do `cp -f` to update existing kubeconfig on later reboots

Reply:
Ack